pages/design/rule-template.md

```diff
@@ -61,12 +61,40 @@ _There are no major accessibility support issues known for this rule._
 
 ## Background
 
-- Links to Techniques for WCAG 2.0
-- Latest version: Techniques for WCAG 2.0 W3C Working Group Note 8 April 2014
-- Link to other methodologies, test specifications or tools
-- Links to test cases, test suites, unit tests, etc.
-- The WCAG 2.0 Techniques already contain examples and code snippets to illustrate which content passes or fails the test. Whenever possible WCAG-ACT-RULES-CG refers to those. Another source for test cases is the W3C Before and After Demonstration.
-- Other references
+<!-- Add an introduction, if you want to add further information not available on the pages you are linking to in this section. -->
```

Comment:
Is there a reason why these comments are only in this template and not in the others?

Reply:
There isn't as far as I see. Good catch!